cost/turbonomics/buy_reserved_instances_recommendations/aws/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+- Initial Release
```

Comment:
Most policies have version 1.0 in their initial release, I would recommend add version number 1.0 above initial release.

https://github.com/flexera-public/policy_templates/blob/f6fc93178e11b80c9e5b1e280156ecbfc57645ba/cost/azure/reserved_instances/expiration/CHANGELOG.md?plain=1

![image](https://user-images.githubusercontent.com/54189123/219091439-3a74464f-c950-4554-9766-2d985ee2e467.png)

Reply:
I remember, Victor Adding 2.0 for the fresh policy. And also I remember Ren updating the same during KT session.

@Rangelmv23 Can you please confirm ?